datalad/customremotes/ria_utils.py

```diff
@@ -203,6 +204,7 @@ def create_ds_in_store(io, base_path, dsid, obj_version, store_version, alias=No
       layout version of the dataset itself (object tree)
     alias: str, optional
       alias for the dataset in the store
+    init_obj_tree: bool
```

Comment:
Worth the additional line to define what `obj_tree` is

Reply:
No idea why I didn't. Thx.